src/components/ContextPanel.tsx

```
@@ -0,0 +1,125 @@
+    <Show when={props.isOpen}>
+      <div
+        class="fixed inset-0 z-40 flex items-center justify-center bg-[var(--color-earthy-dark-brown)]/30 backdrop-blur-sm"
+        role="dialog"
+        aria-modal="true"
+        aria-label="Context and settings"
+        onClick={(e) => e.target === e.currentTarget && props.onClose()}
+      >
+        <div
+          class="w-full max-w-md mx-4 bg-[var(--color-earthy-bg)] rounded-2xl border border-[var(--color-earthy-sage)] shadow-xl overflow-hidden"
+          onClick={(e) => e.stopPropagation()}
+        >
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)